pypeit/wavetilts.py

```diff
@@ -145,6 +168,68 @@ def spatid_to_zero(self, spat_id):
         mtch = self.spat_id == spat_id
         return np.where(mtch)[0][0]
 
+    def show(self, tilt_img=None, slits=None, waveimg=None, wcs_match=True, in_ginga=True, show_traces=False):
+        """
+        """
+        # Warning: This parses the filename, not the Header!
+        master_key, master_dir = masterframe.grab_key_mdir(self.filename, from_filename=True)
```

Comment:
@kbwestfall is refactoring all of this.  hopefully the merge won't be too painful..